gito/gh_api.py

```
@@ -33,3 +42,72 @@
+    mutation($commentId: ID!, $reason: ReportedContentClassifiers!) {
+        minimizeComment(input: {subjectId: $commentId, classifier: $reason}) {
+            minimizedComment { isMinimized }
+        }
+    }"""
+
+    response = requests.post(
+        "https://api.github.com/graphql",
+        headers={"Authorization": f"Bearer {token}"},
+        json={
+            "query": mutation,
+            "variables": {"commentId": comment_id, "reason": reason}
```

Ruff (0.11.9)

Undefined name `comment_id`

(F821)